app/Http/Requests/Staff/StoreDonationTransactionRequest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * NOTICE OF LICENSE.
+ *
+ * UNIT3D Community Edition is open-sourced software licensed under the GNU Affero General Public License v3.0
+ * The details is bundled with this project in the file LICENSE.txt.
+ *
+ * @project    UNIT3D Community Edition
+ *
+ * @author     Roardom <roardom@protonmail.com>
+ * @license    https://www.gnu.org/licenses/agpl-3.0.en.html/ GNU Affero General Public License v3.0
+ */
+
+namespace App\Http\Requests\Staff;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class StoreDonationTransactionRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     */
+    public function authorize(): bool
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, \Illuminate\Contracts\Validation\Rule|array<\Illuminate\Contracts\Validation\Rule|string>|string>
+     */
+    public function rules(): array
+    {
+        return [
+            'item' => [
+                'required',
```

Comment:
```suggestion
                'required',
                'exists:donation_items'
```

We should validate more than the bare minimum.